db/dcrpg/pgblockchain.go

```diff
@@ -5774,7 +5777,7 @@ func (pgb *ChainDB) GetExplorerTx(txid string) *exptypes.TxInfo {
 		valueIn, _ := dcrutil.NewAmount(vin.AmountIn)
 		// Do not attempt to look up prevout if it is a coinbase or stakebase
 		// input, which does not spend a previous output.
-		if !(vin.IsCoinBase() || (vin.IsStakeBase() && i == 0)) {
+		if !(txhelpers.IsCoinbaseVin(vin) || (vin.IsStakeBase() && i == 0)) {
```

Comment:
Looks like you need to do something similar to catch the zeroth treasurybase input here too. e.g. for tx `b35eb58a6bb092c251dc4338d0c1c9891ff7786538e432b777f1d545c2ac0958`.

```
2020/11/21 03:50:27 http: panic serving 127.0.0.1:44776: runtime error: index out of range [0] with length 0
...
github.com/decred/dcrdata/db/dcrpg/v5.(*ChainDB).GetExplorerTx(0xc000144d00, 0xc01e7ef333, 0x40, 0xffffffffffffffff)
	/home/buck/programs/dcrdata/db/dcrpg/pgblockchain.go:5837 +0x1c46
```

But OK with 

```
if !(txhelpers.IsCoinbaseVin(vin) || (txhelpers.IsStakebaseVin(vin) && i == 0) || (i == 0 && standalone.IsTreasuryBase(msgTx))) {
```

Reply:
I had actually started to update the modules and dependencies, including `blockchain/standalone` in this branch: https://github.com/chappjc/dcrdata/commits/nomod

But it looks like in a lot of these checks it's sufficient to detect a null prevout https://github.com/decred/dcrd/blob/master/blockchain/standalone/tx.go#L96